articles/web-application-firewall/waf-sentinel.md

```diff
@@ -79,6 +79,12 @@ To enable log analytics for each resource, go to your individual Azure Front Doo
 1. Once finished configuring individual WAF resources, select the **Next steps** tab. Select one of the recommended workbooks. This workbook will use all log analytic data that was enabled previously. A working WAF workbook should now exist for your WAF resources.
 
    :::image type="content" source="media//waf-sentinel/waf-workbooks.png" alt-text="WAF workbooks" lightbox="media//waf-sentinel/waf-workbooks.png":::
+   
+## Automated Detection and Response for Azure WAF with Sentinel
+
+There are built in detection query templates for SQLi and XSS attacks in Sentinel for Azure WAF. These templates can be found under the Analytics tab in the 'Rule Templates' section of Sentinel. By using these templates, we can create analytic rules that detect specific type of attack patterns in the WAF logs and further notify the security analyst by creating an incident. The automation section of these rules can help us respond to this incident by blocking the source IP of the attacker on the WAF Policy by utilizing a Playbook. This [Playbook](https://github.com/Azure/Azure-Network-Security/tree/master/Azure%20WAF/Playbook%20-%20WAF%20Sentinel%20Playbook%20Block%20IP%20-%20New) can be deployed from the Net Sec Github Repository. 
+
```

Comment:
Remove this para - should not put references to blogs in doc. The blog is already referred in github example.